src/constants/dataTestIds.enum.ts

```diff
@@ -17,4 +17,7 @@ export enum DataTestIdsEnum {
   transactionToastTitle = 'transactionToastTitle',
   transactionDetailsToastBody = 'transactionDetailsToastBody',
   signTransactionBtn = 'signTransactionBtn',
+  signNextTransactionBtn = 'signNextTransactionBtn',
```

Comment:
must be the same as in sdk-dapp today for tests to pass (you can remove the comment)